backend/plugins/posclient_api/src/messageBroker.ts

```
@@ -0,0 +1,376 @@
+        switch (data.type) {
+          case 'product':
+            await receiveProduct(models, data);
+            break;
+          case 'productCategory':
+            await receiveProductCategory(models, data);
+            break;
+          case 'user':
+            await receiveUser(models, data);
+            break;
+          case 'productGroups':
+            const { productGroups = [] } = data;
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)